src/router/kromo/dd-wrt/lang_pack/german.js

```diff
@@ -741,6 +740,7 @@ filter.legend5="Webseite mit Hilfe von Schlagworten blockieren";
 filter.mess1="Löschen der Richtlinie?";
 filter.mess2="Sie müssen mindestens einen Tag angeben.";
 filter.mess3="Die Endzeit muss nach der Startzeit liegen.";
+filter.none="";
```

Comment:
@sparksofinsanity 

This maybe invalid, the latest strings broke the UI.
![Capture](https://user-images.githubusercontent.com/31389848/179391527-7679bcb1-140c-4d4d-80b0-6c225dcdc8df.PNG)

Reply:
1. the string was already before in the english translation.
I have only taken it over for in the German
2. you use the english translation

Reply:
None the less something is breaking English strings, it happens Ive broken strings myself more than once, the problem now is finding where issues is.

General rule of thumb when in a help string you do `<em>" + some.string + "</em>` the `some.string` must be defined before its called from, else it can also cause the issue.

missing " or ; Ive done several times before also break the strings.

No big deal, just need to find where the breakage is happening.

Reply:
If you have time you can look for the error.
I am currently indisposed for the next 60min